### PR TITLE
tutorial improvements

### PR DESCRIPTION
**Pull request by Mateusz512**
align with code
provide examples
some minor changes in text

### CODE REVIEW

File: src/render/tutorials/caching-http-requests/2_0.html.md [hunk 1]
```diff
@@ -254,15 +254,31 @@ The first one is `webapi`
 Then we have `knotx` service specification. It's very similar to the previous one, so only one property is worth explaining. 
 - `command` - this property is used to specify Knot.x starting commands.
 
-Next step in our configuration is to set `rootProject.name` in `settings.gradle.kts` (also at the project level directory),
-also make sure that `healh-check` is included 
-```gradle
-rootProject.name = "api-cache"
+Second container running in our stack will be a container started from `rodolpheche/wiremock`. We will be using Wiremock's [response templating](http://wiremock.org/docs/response-templating/) feature, so the container must be started with appropriate configuration as seen above.
```
Comment (tomaszmichalak) [inline src/render/tutorials/caching-http-requests/2_0.html.md]:
```suggestion
The `webapi` service is the second piece of our environment. It is a flexible API mocking tool, called Wiremock, that serves preconfigured responses for matched requests (based on path and method). It is our target `/product/id` API.
We extend the `rodolpheche/wiremock` image with Wiremock's [response templating](http://wiremock.org/docs/response-templating/) feature. So before we configure the target API behaviour, we need to configure this feature first. So the service must be started with appropriate configuration as seen above.
```

File: src/render/tutorials/caching-http-requests/2_0.html.md [hunk 2]
```diff
@@ -310,7 +330,63 @@ $ gradlew clean build
 $ docker swarm init
 $ docker stack deploy -c api-cache.yml api-cache
 ```
-After successful execution you should be able to see http response at [localhost:8092/product/id](localhost:8092/product/id)
+After successful execution you should be able to see http response at [`localhost:3000/product/id`](localhost:3000/product/id). This is te response served by Wiremock.
```
Comment (tomaszmichalak) [inline src/render/tutorials/caching-http-requests/2_0.html.md]:
```suggestion
After successful deployment, when opening the URL [`localhost:3000/product/id`](localhost:3000/product/id), you should receive the HTTP 200 response served by Wiremock.
```

File: src/render/tutorials/caching-http-requests/2_0.html.md [hunk 3]
```diff
@@ -310,7 +330,63 @@ $ gradlew clean build
...
+}
+> curl -X GET http://localhost:3000/product/id
+{
+  "id": 21762532,
+  "url": "http://knotx.io",
+  "label": "Product",
+  "currentTime": "2019-10-29 13:57:57",
+  "promoCode": "ymhvkg"
+}
+> curl -X GET http://localhost:3000/product/id
```
Comment (tomaszmichalak) [inline src/render/tutorials/caching-http-requests/2_0.html.md]: Two examples are enough.

File: src/render/tutorials/caching-http-requests/2_0.html.md [hunk 4]
```diff
@@ -310,7 +330,63 @@ $ gradlew clean build
...
+}
+> curl -X GET http://localhost:8092/product/id
+{
+  "id" : 21762532,
+  "url" : "http://knotx.io",
+  "label" : "Product",
+  "currentTime" : "2019-10-29 13:58:09",
+  "promoCode" : "y8v7u2"
+}
+> curl -X GET http://localhost:8092/product/id
```
Comment (tomaszmichalak) [inline src/render/tutorials/caching-http-requests/2_0.html.md]: Two examples are enough.

File: src/render/tutorials/caching-http-requests/2_0.html.md [hunk 5]
```diff
@@ -310,7 +330,63 @@ $ gradlew clean build
 $ docker swarm init
 $ docker stack deploy -c api-cache.yml api-cache
 ```
-After successful execution you should be able to see http response at [localhost:8092/product/id](localhost:8092/product/id)
+After successful execution you should be able to see http response at [`localhost:3000/product/id`](localhost:3000/product/id). This is te response served by Wiremock. 
+
+Now if you invoke Knot.x at [`localhost:8092/product/id`](localhost:8092/product/id) and refresh it a few times, you can see that neither the timestamp nor the random string in the JSON response changes - we have successfully cached external API response!
+
+```
+> curl -X GET http://localhost:3000/product/id
```
Comment (tomaszmichalak) [inline src/render/tutorials/caching-http-requests/2_0.html.md]: Maybe lets move those responses before:
> No if...

File: src/render/tutorials/caching-http-requests/2_0.html.md (hunk 1 above)
Comment (Mateusz512) [reply src/render/tutorials/caching-http-requests/2_0.html.md]: done

File: src/render/tutorials/caching-http-requests/2_0.html.md (hunk 2 above)
Comment (Mateusz512) [reply src/render/tutorials/caching-http-requests/2_0.html.md]: done

File: src/render/tutorials/caching-http-requests/2_0.html.md (hunk 4 above)
Comment (Mateusz512) [reply src/render/tutorials/caching-http-requests/2_0.html.md]: done

File: src/render/tutorials/caching-http-requests/2_0.html.md (hunk 3 above)
Comment (Mateusz512) [reply src/render/tutorials/caching-http-requests/2_0.html.md]: done

File: src/render/tutorials/caching-http-requests/2_0.html.md (hunk 5 above)
Comment (Mateusz512) [reply src/render/tutorials/caching-http-requests/2_0.html.md]: done

**Pull request merged**